cob_moveit_config/robots/raw3-1/moveit/config/constraints_limit.yaml

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
where is this file used?
please remove this file if it was also only used for testing purposes as in *_calib...